PassleSync.Core/Controllers/PassleDashboard/PassleDashboardSettingsController.cs

```diff
@@ -35,8 +38,10 @@ public PassleDashboardSettingsViewModel Get()
                 PassleShortcodes = _configService.PassleShortcodesString,
                 ClientApiKey = _configService.ClientApiKey,
                 PluginApiKey = _configService.PluginApiKey,
-                PostPermalinkPrefix = _configService.PostPermalinkPrefix,
-                AuthorPermalinkPrefix = _configService.AuthorPermalinkPrefix,
+                PostPermalinkTemplate = _configService.PostPermalinkTemplate ?? "p",

```

Comment:
Should the default include the variables?
e.g. I think we generally use `p/{{PostShortcode}}/{{PostSlug}}` as a default
Same applies below